### PR TITLE
Update media_cleaner.py

### PR DESCRIPTION
**Pull request by terrelsa13**
-Use plain text password one time to get the access token.
    -Access token is now stored.
    -Password is no longer stored.
-Add not_played_age_* configs for movies, episodes, videos, and trailers to allow different days for each media type.
    -Remove '*_action' configs
    -(-1) in not_played_age_* will disable deleting each media type.
-Attempt at updating output and putting into sections.
    -Showing 'days since watched' for each watched movie, episode, video, and trailer.
    -Add series name to episode output.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,5 +30,3 @@ Make media_cleaner.py executable and run ./media_cleaner.py.  If no conifg file
 
 # Donation
 If you find this useful and you would like to support please the use option below.
-
-[![paypal](https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif)](https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=jason%2ep%2eclara%40gmail%2ecom&lc=CA&item_name=Jason%20Clara&currency_code=USD&bn=PP%2dDonationsBF%3abtn_donateCC_LG%2egif%3aNonHosted)
```
Comment (terrelsa13) [inline README.md]: Accidental delete of the donation information. Added back in subsequent commit.

**Comment by clara-j** [comment]
Great work on this, I had been slowly working on cleaning up the code, but haven't had too much time.  So this is very helpful.

One thing I just want to look up before I accept the pull is the access token vs storing the hashed password.  The reason I went with the hashed password vs the token was that I was under the impression that the token would expire.  So that is why I stored the hashed password and then get a new token each time.

**Comment by terrelsa13** [comment]
No problem.

This is a [recent post](https://emby.media/community/index.php?/topic/78948-login-as-a-user-with-admin-token/?hl=authenticatebyname) on the emby forum I found that lead me to believe the tokens do not expire. But I could be wrong. If it turns out tokens do expire we can add storing and using the hashed password back into the script.

**Comment by terrelsa13** [comment]
Another possible option:
If it turns out the API generated access token does expire; a token can be manually created via the emby server API GUI. The script would then prompt the user to copy/paste the manual created token. This token would never expire (unless revoked) and a password would not have to be input or stored.

**Comment by clara-j** [comment]
I had tried with the generated API key, but for some reason I couldn't get it to have permissions to delete.

I can't find anything showing the token expires so I will merge and revert back if it causes any problems.  

Thanks again for the work on this.

**Pull request merged**